src/js/components/common/ModalRescuer.tsx

```diff
@@ -14,7 +14,7 @@ export type ModalRescuerProps = {
 const ModalRescuer = ({ isOpen, close }: ModalRescuerProps) => {
   useBlur(isOpen);
   return (
-    <Popup open={isOpen} closeOnDocumentClick onClose={close}>
+    <Popup open={isOpen} closeOnDocumentClick onClose={close} modal nested>
       <Card.Card>
```

Comment:
Zastanawiam się czy chcemy żeby zawsze był to modal, czy samo nested nie wystarczy?

Reply:
W sumie wystarczy, mogę usunąć `modal`